Index: clisp-2.49/tests/tests.lisp
===================================================================
--- clisp-2.49.orig/tests/tests.lisp
+++ clisp-2.49/tests/tests.lisp
@@ -381,7 +381,7 @@ NIL: sacla-style: forms should evaluate
     #+(or XCL CLISP)  ("path")
     #+XCL             ("readtable")
     #-CMU             ("setf")
-    #+(and CLISP SOCKETS) ("socket")
+;    #+(and CLISP SOCKETS) ("socket")
                       ("steele7")
     #-ALLEGRO         ("streams")
                       ("streamslong")
